contract-service/src/main/java/com/example/contractservice/common/util/feign/CommissionClient.java

```diff
@@ -0,0 +1,13 @@
+package com.example.contractservice.common.util.feign;
+
+import com.example.contractservice.contract.service.dto.response.CommissionRecruitmentResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(name = "commissionClient", path = "internal/commissions")
```

Comment:
`internal` 앞에 `/`를 생략해도 괜찮은지 궁금합니다😯

```suggestion
@FeignClient(name = "commissionClient", path = "/internal/commissions")
```

Reply:
오 안될 것 같네요. 감사합니다.